build_runner/lib/src/package_graph/build_config_overrides.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:build_config/build_config.dart';
+import 'package:glob/glob.dart';
+import 'package:path/path.dart' as p;
+
+import 'package_graph.dart';
+
+Future<Map<String, BuildConfig>> findBuildConfigOverrides(
+    PackageGraph packageGraph) async {
+  final configs = <String, BuildConfig>{};
+  final configFiles = await new Glob('*.build.yaml').list().toList();
+  for (final file in configFiles) {
```

Comment:
nit: keep the stream and use `await for ...`

Reply:
Done